sqlglot/generator.py

```diff
@@ -3860,15 +3860,16 @@ def _replace_line_breaks(self, string: str) -> str:
     def copyparameter_sql(self, expression: exp.CopyParameter) -> str:
         option = self.sql(expression, "this")
 
-        upper = option.upper()
-        if upper == "FILE_FORMAT":
+        if isinstance(expression.expression, list):
```

Comment:
This shouldn't be a list if the argument is singular.

Reply:
Some of the options are singular and some are list, but the distinction between the two is helpful for us so we can change the output. Should I rename the argument to `expressions` but keep the semantics of `t.List` vs `t.Expression` ?

Reply:
Or I can add `expressions` alongside `expression` but one of the two arguments will always be empty, although not an issue

Reply:
We should add a new `expressions` argument so these two cases are distinct. Having a single one to represent both cases is confusing and deviates from the codebase style.

Reply:
> one of the two arguments will always be empty

This is fine, we do it all over the place anyway right?

Reply:
Right, but I was thinking that each COPY will have a (long ?) list of `exp.CopyParameters` so maybe it's more wasteful than other exprs that are created once. Still shouldn't make any difference